src/hotspot/share/utilities/compilerWarnings_gcc.hpp

```diff
@@ -44,6 +44,12 @@
 
 #define PRAGMA_FORMAT_IGNORED PRAGMA_DISABLE_GCC_WARNING("-Wformat")
 
+#define PRAGMA_FORMAT_OVERFLOW_IGNORED \
+  PRAGMA_DISABLE_GCC_WARNING("-Wformat-overflow")
+
+#define PRAGMA_STRINGOP_OVERFLOW_IGNORED \
+  PRAGMA_DISABLE_GCC_WARNING("-Wstringop-overflow")
```

Comment:
Are the reported problems with format/stringop-overflow real? Or are they
false positives? If they are real then I'm not going to approve ignoring them,
unless there is some urgent reason and there are followup bug reports for
fixing them.

Reply:
I think all of warnings in HotSpot are false-positives, so I added new pragmas to compilerWarnings.hpp, and use it in HotSpot code.